server/README.md

```diff
@@ -101,6 +101,18 @@ make run-api
 
 And open http://localhost:8080/v1/ui
 
+If you want to run your own API image, use instead:
+```
+# Run the API container
+IMAGE=your_api_image:tag make run-api
+```
+
+You can clean API data, created from `make run-api`, by running:
```

Comment:
Hm, I bet you changed this line in the old PR...

Reply:
yep, and I forgot to pull before opening this PR.
Done.